java/test/src/main/java/io/ray/test/PlacementGroupTest.java

```diff
@@ -142,4 +147,73 @@ public void testBundleSizeValidCheckWhenCreate() {
   public void testBundleResourceValidCheckWhenCreate() {
     PlacementGroupTestUtils.createBundleResourceInvalidGroup();
   }
+
+  @Test(groups = {"cluster"})
+  public void testNamedPlacementGroup() {
+    // Test Non-Global placement group.
+    String pgName = "named_placement_group";
+    PlacementGroup firstPlacementGroup =
+        PlacementGroupTestUtils.createNameSpecifiedSimpleGroup(
+            "CPU", 1, PlacementStrategy.PACK, 1.0, pgName, false);
+    Assert.assertTrue(firstPlacementGroup.wait(60));
+    // Make sure we can get it by name successfully.
+    PlacementGroup placementGroup = Ray.getPlacementGroup(pgName);
+    Assert.assertNotNull(placementGroup);
+    Assert.assertEquals(placementGroup.getBundles().size(), 1);
+
+    // Test global placement group.
+    String pgGlobalName = "global_placement_group";
+    PlacementGroup secondPlacementGroup =
+        PlacementGroupTestUtils.createNameSpecifiedSimpleGroup(
+            "CPU", 1, PlacementStrategy.PACK, 1.0, pgGlobalName, true);
+    Assert.assertTrue(secondPlacementGroup.wait(60));
+    // Make sure we can get it by name successfully.
+    placementGroup = Ray.getGlobalPlacementGroup(pgGlobalName);
+    Assert.assertNotNull(placementGroup);
+    Assert.assertEquals(placementGroup.getBundles().size(), 1);
+  }
+
+  @Test(
+      expectedExceptions = {IllegalArgumentException.class},
+      groups = {"cluster"})
+  public void testCreatePlacementGroupWithSameName() {
+    String pgName = "named_placement_group";
+    PlacementGroup firstPlacementGroup =
+        PlacementGroupTestUtils.createNameSpecifiedSimpleGroup(
+            "CPU", 1, PlacementStrategy.PACK, 1.0, pgName, false);
+    Assert.assertTrue(firstPlacementGroup.wait(60));
+    PlacementGroupTestUtils.createNameSpecifiedSimpleGroup(
+        "CPU", 1, PlacementStrategy.PACK, 1.0, pgName, false);
+  }
```

Comment:
No verification?

Reply:
Yes, because this is an exceptional case and the second creation will be failed.